build/shimplify.py

```diff
@@ -0,0 +1,555 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Implementation of <shimplify> Ant task in Python2.7 for Jython
+
+Simplifies the prior version-range directory system
+
+The if=false version is run by default during the generate-sources phase. If the user is
+to run shimplify to perform file modifications for converting to the new system or to add a new
```

Comment:
nit: this and other references of shimplify in documentation should be capitalized to Shimplify. Same with Shimple and other derivatives of Shimplify.

Reply:
Do you mean shimple?! 😄